events-api/src/modules/events/events.repository.ts

```diff
@@ -18,4 +20,62 @@ export class EventsRepository {
 
     return result;
   }
+
+  static async findById(id: string): Promise<EventModel[] | null> {
+    const res = await db.execute<EventModel>(
+      `SELECT event_id FROM events WHERE event_id = ?`,
```

Comment:
Why do you select only event_id?

Reply:
Also in EventModel, there is no event_id property.

Reply:
done